sgpt/app.py

```diff
@@ -57,7 +57,7 @@ def main(
         rich_help_panel="Assistance Options",
     ),
     interaction: bool = typer.Option(
-        True,
+        True if cfg.get("SHELL_INTERACTION") != "false" else False,
```

Comment:
```suggestion
        cfg.get("SHELL_INTERACTION") == "true",
```

Reply:
I wrote it this way to ensure the normal default behaviour is set, also when SHELL_INTERACTION contains an unintended / weird value

Reply:
In order to keep the code base consistent, I would like to reuse existing patters as we checking the boolean values few lines above.